src/luascript.cpp

```diff
@@ -10717,6 +10717,11 @@ int LuaScriptInterface::luaItemTypeCreate(lua_State* L)
 		id = Item::items.getItemIdByName(getString(L, 2));
```

Comment:
You should add the check above this line instead. If string is empty, don't even call the function.

Reply:
Yes I know, but `uint16_t Items::getItemIdByName(const std::string& name)` should not return empty name item too.